util/admin/admin.go

```diff
@@ -287,6 +288,24 @@ func getIndexFieldTypes(t table.Table, idx table.Index) ([]*types.FieldType, err
 	return fieldTypes, nil
 }
 
+// we should treat KindString equals to KindBytes.
```

Comment:
This sentence is inappropriate in grammar. Pls choose the one that you want to express:

- This function treats KindString and KindBytes in the same way.
- This function treats KindString as KindBytes.